docs/src/list_of_supported_models.md

```diff
@@ -29,7 +29,7 @@ independent assessment.
 | ------- | ------ | -------- | ----
 [BetaML.jl](https://github.com/sylvaticus/BetaML.jl) | DecisionTreeClassifier, DecisionTreeRegressor, KernelPerceptronClassifier, PegasosClassifier, PerceptronClassifier, RandomForestClassifier | medium |
 [Clustering.jl](https://github.com/JuliaStats/Clustering.jl) | KMeans, KMedoids | high | †
-[DecisionTree.jl](https://github.com/bensadeghi/DecisionTree.jl) | DecisionTreeClassifier, DecisionTreeRegressor, AdaBoostStumpClassifier, RandomForestClassifier, RandomForestRegressor | high | 
+[DecisionTree.jl](https://github.com/bensadeghi/DecisionTree.jl) | DecisionTreeClassifier, DecisionTreeRegressor, AdaBoostStumpClassifier, RandomForestClassifier, RandomForestRegressor | high | also via ScikitLearn.jl and no longer maintained
```

Comment:
@zsz00 Thanks for looking at this. 

It's true that DecisionTree.jl implements the ScikitLearnBase.jl API. However, the package provides a pure Julia implementation of the core algorithm (and is not wrapping any python code). Also, the MLJ interface is completely orthogonal to the ScikitLearnBase.jl API. So I think "also via ScikitLearn.jl" is not accurate. 

I see now the author has indeed marked the package as no longer maintained, which is a pity. So that is fair comment.

Reply:
Ok, I don't check the core implementation. I removed "also via ScikitLearn.jl"